staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go

```diff
@@ -121,7 +121,10 @@ func (o *ReconcileOptions) Complete(cmd *cobra.Command, f cmdutil.Factory, args
 		return errors.New("no arguments are allowed")
 	}
 
-	o.DryRun = getClientSideDryRun(cmd)
+	var err error
```

Comment:
hey, just a small nit here:

What about moving this var to be the first call of the function Complete(), and in https://github.com/kubernetes/kubernetes/pull/97645/files#diff-b72235b951dc6695431906e118b1eaf840673897cec491cc528025b1ff0f2a2bR116 remove the new assignment of the variable (instead of err :=, keep err =)

This way, the declared err variable can be reused across all the future calls :)

Reply:
tks, fixed.